content/en/metrics/nested_queries.md

```diff
@@ -0,0 +1,165 @@
+
+```text
+sum(avg:aws.ec2.cpuutilization{*} by {env,host}.rollup(avg, 300),{env})
+```
+
+For percentile space aggregation, the following query calculates the 95th percentile of average CPU utilization, grouped by environment:
+
+```text
+percentile(avg:aws.ec2.cpuutilization{*} by {env,host}.rollup(avg, 300),'p95', {env})
+```
+In the UI or JSON tab, it would look as follows:
+
+{{< img src="/metrics/nested_queries/multilayer-space-agg-ui.png" alt="example of multilayer space aggregation in the UI" style="width:100%;" >}}
+
+{{< img src="/metrics/nested_queries/multilayer-space-agg-json.png" alt="example of multilayer space aggregation in the JSON" style="width:100%;" >}}
+{{% /collapse-content %}} 
+
+
+
+## Percentiles
+
+Percentile calculations provide deeper insights into your data's distribution by allowing you to set performance thresholds, and identifying outliers/extremes in your data.
+
+Here's an example that calculates the 95th percentile of average CPU utilization, grouped by environment and host, rolled up into 5-minute intervals, over the last 30 minutes:
+
+{{% collapse-content title="Percentiles example query" level="h5" %}}
+```text
+"rollup(avg:aws.ec2.cpuutilization{*} by {env,host}.rollup(avg, 300),'p95',1800)"
+```
+
+In the UI or JSON tab, it would look as follows:
+
+ {{< img src="/metrics/nested_queries/nested-queries-percentiles-ui.png" alt="example of percentiles  using nested queries in the UI" style="width:100%;" >}}
+
+ {{< img src="/metrics/nested_queries/nested-queries-percentiles-json.png" alt="example of percentiles  using nested queries in the JSON" style="width:100%;" >}}
+{{% /collapse-content %}} 
+
+
+
+## Standard Deviation
+
+Standard deviation helps measure the variability or dispersion of a dataset. The following query calculates the standard deviation of the sum of API request counts, averaged over 5-minute intervals, and rolled up over 1-hour periods:
+
+{{% collapse-content title="Standard deviation example query" level="h5" %}}
+```text
+"rollup(sum:api.requests.count{*}.rollup(avg,300),'stddev',14400)"
+```
+In the UI or JSON tab, it would look as follows:
+
+ {{< img src="/metrics/nested_queries/nested-queries-std-ui.png" alt="example of standard deviation with nested queries in the UI" style="width:100%;" >}}
+
+ {{< img src="/metrics/nested_queries/nested-queries-std-jsonigh.png" alt="example of standard deviation with nested queries in the JSON" style="width:100%;" >}}
+{{% /collapse-content %}} 
+
+
+
+## Higher Resolution Queries
+
+As the query time frame increases, the rollup interval becomes less granular, preventing you from querying anything more detailed than those default time intervals. With nested queries, you can now access more granular, high-resolution data over longer periods.
+
+Here's an example that calculates the standard deviation of high-resolution metrics batch counts, with a 5-minute base and a 4-hour reducer interval:
```

Comment:
```suggestion
```

Reply:
Since we don't really mention the term reducer anywhere above, I'd recommend not introducing terms that aren't defined in this summary sentence